packages/InteractiveProfilingTool-Core.package/IPTModel.class/instance/getMessageTally.st

```diff
@@ -0,0 +1,10 @@
+profiling
+getMessageTally
```

Comment:
The prefix `get` makes the method seem like a simple accessor, however this is not the case. Please consider renaming this method. Suggestion: `spyOnWorld`